src/twisted/test/test_failure.py

```diff
@@ -22,10 +22,7 @@
 from twisted.trial.unittest import SynchronousTestCase
 
 
-try:
-    from twisted.test import raiser  # type: ignore[attr-defined]
-except ImportError:
-    raiser = None
+from cython_test_exception_raiser import raiser
```

Comment:
Maybe it is worth adding a short comment here that this is a test helper which is hosted outside of the main twisted repo...or just a link to the repo.

Reply:
My text editor already does this (go to definition)

Reply:
there is no such feature in mousepad :)... it ok. I don't expect this code will be read soon. so don't worry

Reply:
one questions...  is cython also supported on PYPY? I see that the ImportError is no longer handled and that the test on pypy is green.

how come?